main/main.c

```diff
@@ -611,6 +611,7 @@ PHP_INI_BEGIN()
 #ifdef PHP_WIN32
 	STD_PHP_INI_BOOLEAN("windows.show_crt_warning",		"0",		PHP_INI_ALL,		OnUpdateBool,			windows_show_crt_warning,			php_core_globals,	core_globals)
 #endif
+	STD_PHP_INI_ENTRY("syslog.highbit",			"0",		 PHP_INI_SYSTEM,	OnUpdateBool,		syslog_highbit,		php_core_globals, 		core_globals)
```

Comment:
> This solves the initial issue and puts in some security bits.

@weltling Well, it puts them in but largely disables them.  Anyway, I don't care.  All I ever wanted was splitting stack traces in the exception handler so they wouldn't look like:

```
Aug 11 15:19:01 ubuntu16 php: PHP Fatal error:  Uncaught Error: Class 'NoSuchClass' not found in /home/philipp/bertram/php-src/test.php:4#012Stack trace:#012#0 /home/philipp/bertram/php-src/test.php(15): foo()#012#1 {main}#012  thrown in /home/philipp/bertram/php-src/test.php on line 4
```

which is what they currently look like.  I've spent 6x more time on this issue than initially planned.

We've still not decided if DEL should ever get written in the clear or not.  Nothing I've seen suggests that 0xf7 should be allowed.